mllib/src/main/scala/org/apache/spark/mllib/feature/PCA.scala

```diff
@@ -68,7 +68,7 @@ class PCA(val k: Int) {
  * @param k number of principal components.
  * @param pc a principal components Matrix. Each column is one principal component.
  */
-class PCAModel private[mllib] (val k: Int, val pc: DenseMatrix) extends VectorTransformer {
+class PCAModel private[spark] (val k: Int, val pc: DenseMatrix) extends VectorTransformer {
```

Comment:
Because test case of ml.feature.PCASuite(https://github.com/apache/spark/pull/7065/files#diff-e1593bb9e311c3f2a2ea49cce20ed671R34) use the constructor, so I change it to spark private like Word2VecModel.
There are different access permission of constructors in mllib.feature, some are private[spark] while others are public. I think it's confusion and need to uniform in a separate task.